thread.c

```diff
@@ -5374,9 +5384,6 @@ Init_Thread(void)
 	    /* acquire global vm lock */
 	    gvl_init(th->vm);
 	    gvl_acquire(th->vm, th);
```

Comment:
I'm not sure if there is significance in initializing the GVL before all these other mutexes are initialized. If it is not important, then the current change should be fine.